src/components/block-css/index.js

```diff
@@ -433,7 +441,8 @@ const BlockCssEdit = props => {
 		instanceId,
 	} )
 
-	const output = useDynamicContent( css )
+	let output = useDynamicContent( css )
+	output = applyFilters( 'stackable.block-styles.edit', output, getUniqueBlockClass( attributes.uniqueId ) )
```

Comment:
Same here, you most probably forgot to also pass the attributes as a filter argument.